packages/neos-ui-redux-store/src/CR/Nodes/index.js

```diff
@@ -95,11 +94,10 @@ export const reducer = handleActions({
             siteNode: $get('cr.nodes.siteNode', state) || '',
             focused: new Map({
                 contextPath: '',
-                typoscriptPath: ''
+                fusionPath: ''
```

Comment:
add a comment here why fusionPath is needed (e.g. only for out-of-band-rendering when adding new nodes)